includes/admin/class-edd-notices.php

```diff
@@ -103,6 +103,74 @@ public function show_notices() {
 
 		}
 
+		/**
+		 * Notice for users running PHP < 5.6.
+		 * @since 2.10
+		 */
+		if ( version_compare( PHP_VERSION, '5.6', '<' ) && ! get_user_meta( get_current_user_id(), '_edd_upgrade_php_dismissed', true ) && current_user_can( 'manage_shop_settings' ) ) {
+			echo '<div class="notice notice-warning is-dismissible edd-notice">';
+			printf(
+				'<h2>%s</h2>',
+				esc_html__( 'Get Ready for Easy Digital Downloads 3.0', 'easy-digital-downloads' )
```

Comment:
I wonder if this is clear enough that "you need to do something". My first impression leaned more towards "exciting update is coming! Read about the features" as opposed to "you have to take action".

Reply:
We could change it to something like: "Easy Digital Downloads 3.0: Your Site Is Not Ready" ... "Don't Miss Out On Easy Digital Downloads 3.0" ... ?

Reply:
I was thinking something like _"Upgrades Required for Easy Digital Downloads 3.0"_ but then I wondered if that sounds like you've been upgraded to 3.0 and it wants to do a DB upgrade. :thinking:

Reply:
It's tough because something like "Action Required" (or "Upgrades Required") isn't _technically_ true since it doesn't need to be done immediately... but I think it should learn towards being too aggressive vs. too passive.

"Action Required" is a pretty frequent subject line in my mailbox:

> Action required: Changes to the Events API that affect your team's Slack apps

> [Action Required] Automated Certificate Management and compatibility with Android versions prior to v.7.1.1

> Action Required: Migrate to the new SmartThings app

> Action Required: Confirm Your Email Now

So maybe something like:

**Action Required: Upgrade PHP to Prepare for Easy Digital Downloads 3.0**

---

It would be interesting to show an additional notice when the action has actually been completed. 

1. See action require/upgrade required notice
2. Perform action off-site
3. Return to site and see new notice confirming no additional steps are needed

Reply:
> Upgrade PHP to Prepare for Easy Digital Downloads 3.0

I like that.

> It would be interesting to show an additional notice when the action has actually been completed.

I'm not necessarily opposed to that, but do question if it's worth the extra effort. I also wonder if that's getting too notice happy. :shrug:

Reply:
> I'm not necessarily opposed to that, but do question if it's worth the extra effort. I also wonder if that's getting too notice happy. 🤷

Worth questioning for sure -- and you may be right. I just like the idea of having confirmation since it's not something we can fully walk them through. But it might not be worth it for the few lingering installs. People should have been getting upgrade notices for years at this point.

Reply:
Yeah given that it's the minority of users, it might actually be easier/faster to just handle those in support for anyone who is unsure or wants confirmation. A support person can tell immediately from the system info file if they're good to go.